backbone.d3view.js

```diff
@@ -130,7 +153,7 @@
 
       for (var eventName in map) {
         _.each(map[eventName], function(handler) {
-          removeEvent(this.d3el, eventName, handler.selector, handler.namespace);
+          removeEvent(this.d3el, eventName, handler.selector);
```

Comment:
`removeEvent` should simply be a call to `removeEventListener`, no? We're not using D3's event management at all...